### PR TITLE
Update terraform

### PR DESCRIPTION
**Pull request by jparra5**

### CODE REVIEW

**Comment by jparra5** [comment]
@jauninb

File: cra/task-terraform-scan.yaml [hunk 1]
```diff
@@ -13,6 +13,8 @@ spec:
     - name: branch
       description: the branch
       default: master
+    - name: commit-id
+      description: git commit id
```
Comment (jauninb) [inline cra/task-terraform-scan.yaml:17]: Existing user/adopter of this task will be broken as this commit-id parameter is a new expected parameter for this task
So in order to be previous-versio compliant a default value should be defined (asusming "") and take in account by the underlying node program

**Review by jauninb — CHANGES_REQUESTED** [review]
commit-id new parameter is abreaking change - default value needed

**Comment by jparra5** [comment]
@jauninb @padraic-edwards The comment has been resolved.

**Review by jauninb — APPROVED** [review]
LGTM

**Pull request merged**